### ISSUE TITLE
create render fall inside an infinite loop

### ISSUE THREAD

**Issue by vincenzopalazzo** [anvaka/ngraph.pixel#19]
I try to build a straightforward svelte app with some code examples, but when I add a new link, my code will be stuck in rendering (or try to render) the graph.

This may be related to this exception that I see. 

```
REE.BufferGeometry.computeBoundingSphere(): Computed radius is NaN. The "position" attribute is likely to have NaN values. TH
```

Any idea?

The repo, when you can reproduce it is there https://git.hedwing.dev/vincenzopalazzo/ngraph-svelte

**Comment by anvaka** [comment]
Yeah, that error seems like a decent smoke pointing in the right direction. 

I didn't look close into the issue yet, but are you able to open dev tools, then navigate to the offending page and then hit the pause button to see the stack trace where it is stuck?

**Comment by anvaka** [comment]
Glad you figured it out!

**Comment by anvaka** [comment]
Yeah, looking at it closer, it seems like this module uses  https://github.com/anvaka/pixel.layout/blob/master/index.js which is very old.

Probably should just drop it and switch to ngraph.forcelayout like you did.

Also note - the rendering in ngraph.pixel is done by THREE.js, which I don't use frequently these days. More lightweight options are available. This is my typical go-to starting pack for graph drawing: https://github.com/anvaka/graph-start

**Comment by vincenzopalazzo** [comment]
Hey @anvaka 

>Probably should drop it and switch to ngraph.forcelayout like you did.

Yeah I was pretty sure that also the module was using the forcelayour, but I was wrong

>Also note - the rendering in ngraph.pixel is done by THREE.js, which I don't use frequently. More lightweight options are available. This is my typical go-to starting pack for graph drawing: https://github.com/anvaka/graph-start

I noted it, but it looks like there is a `ngraph.wgl` package, and I am a pretty dum person with the graphics.

Maybe I can start one, and you can supervise it?

**Comment by Josh-Cena** [comment]
Hit the same error and forcedlayout worked great! Took me a while to figure out the options since these aren't documented.
